tests/Doctrine/Tests/DbalFunctionalTestCase.php

```diff
@@ -17,12 +17,12 @@ class DbalFunctionalTestCase extends DbalTestCase
     /**
      * Shared connection when a TestCase is run alone (outside of it's functional suite)
      *
-     * @var \Doctrine\DBAL\Connection
+     * @var \Doctrine\DBAL\Connection|null
      */
     private static $_sharedConn;
 
     /**
-     * @var \Doctrine\DBAL\Connection
+     * @var mixed
```

Comment:
Please revert.

Reply:
It breaks a lot of places:

```
 ------ ------------------------------------------------------------------------------------ 
  Line   tests/Doctrine/Tests/DBAL/Functional/Driver/IBMDB2/DB2StatementTest.php             
 ------ ------------------------------------------------------------------------------------ 
  32     Call to an undefined method Doctrine\DBAL\Driver\Statement::getWrappedStatement().  
 ------ ------------------------------------------------------------------------------------ 

 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/Doctrine/Tests/DBAL/Functional/Driver/OCI8/OCI8ConnectionTest.php                                                                                                                
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  29     Property Doctrine\Tests\DBAL\Functional\Driver\OCI8\OCI8ConnectionTest::$driverConnection (Doctrine\DBAL\Driver\OCI8\OCI8Connection) does not accept Doctrine\DBAL\Driver\Connection.  
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/Doctrine/Tests/DBAL/Functional/Driver/PDOConnectionTest.php                                                                                                          
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  27     Property Doctrine\Tests\DBAL\Functional\Driver\PDOConnectionTest::$driverConnection (Doctrine\DBAL\Driver\PDOConnection) does not accept Doctrine\DBAL\Driver\Connection.  
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
 ------ ------------------------------------------------------------------------------ 
  Line   tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL630Test.php                   
 ------ ------------------------------------------------------------------------------ 
  41     Call to an undefined method Doctrine\DBAL\Driver\Connection::setAttribute().  
  75     Call to an undefined method Doctrine\DBAL\Driver\Connection::setAttribute().  
  99     Call to an undefined method Doctrine\DBAL\Driver\Connection::setAttribute().  
  123    Call to an undefined method Doctrine\DBAL\Driver\Connection::setAttribute().  
 ------ ------------------------------------------------------------------------------ 
```

Reply:
Those breakages look valid though, looks PDO related. Explicitly `mixed` only disables some checks (so better to avoid `mixed`).